src/lib.rs

```diff
@@ -279,9 +295,14 @@ impl<S: Subscriber + for<'a> LookupSpan<'a>> Layer<S> for WASMLayer {
                     .file()
                     .and_then(|file| meta.line().map(|ln| format!("{}:{}", file, ln)))
                     .unwrap_or_default();
+                let timestamp = if self.config.log_time {
+                    chrono::Local::now().format("%a %d %T%.f").to_string()
+                } else {
+                    String::new()
+                };
```

Comment:
If we're in certain WASM contexts where `Date` is available, I wonder if we could just leverage `(new Date()).toSomeFormatString()`. This may allow us to drop `chrono` entirely.

Just thinking out loud.

@llogiq might have other ideas.

Reply:
there is js_sys which exposes the [`Date`](https://docs.rs/js-sys/0.3.45/js_sys/struct.Date.html) APIi. Although I think it is not very future proof, as most Rust crates use `std::time` or `chrono`. I think `js_sys` is even used [internally](https://github.com/chronotope/chrono/blob/main/src/datetime.rs#L795) by `chrono` to get the time in wasm.

Reply:
As I wrote above, I doubt that chrono is increasing our compile times too much, and with javascript methods I'm always wary about the implementation (though nowadays runtimes are usually very well implemented, perhaps that's just the cranky old man in me speaking :smile:).